tox.ini

```diff
@@ -28,7 +28,7 @@ deps =
     django20: Django>=2.0,<2.1
     -r{toxinidir}/requirements/test.txt
 commands =
-    py.test {posargs}
+    py.test --nomigrations {posargs}
```

Comment:
pytest doesn't like running model code during migrations, so it chokes on L34 of the migration.  There's probably a better way to resolve it than skipping migrations, but that is lower priority than this PR.